app/clients/sms/firetext.py

```diff
@@ -9,9 +9,32 @@
 
 logger = logging.getLogger(__name__)
 
+firetext_response_status = {
```

Comment:
Firetext status. Mapped to notify status. Note "undelivered/pending" is an odd state and not sure what we should store that as, if we'd get another callback to update to delivered. Again a question for Firetext. (sent).